src/Compiler/Service/service.fs

```diff
@@ -685,7 +685,7 @@ type BackgroundCompiler
                     "BackgroundCompiler.CheckFileInProject"
                     [|
                         "project", options.ProjectFileName
-                        "fileName", fileName
+                        Activity.Tags.fileName, fileName
```

Comment:
We have these/similar tag names in many more places - are you planning to unify them all in this PR, or was this just an example of what's to come?

Reply:
Will unify to make sure we use them, yes.